scripts/create_sample_ticket.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+echo "c9a5c85d-7d8a-4f8c-8424-ba7f6434d010,JT-Line,Day,Adult,2020-06-11T10:48:44+03:00,2020-06-12T03:00:00+03:00"$'\r' >> $(dirname $0)/../tickets.csv
```

Comment:
What does `$'\r'` do at the end of the line?

Reply:
Added some description in c2c09af49501b42f5c3d7045dea368f15214cd3c